erudit/core/subscription/models.py

```diff
@@ -35,7 +36,7 @@ class Meta:
         verbose_name = _('Revue')
 
 
-class IndividualAccount(models.Model):
+class IndividualAccount(User):
```

Comment:
Je modifierais l'implémentation du `IndividualAccount`. Ailleurs dans le projet, on a une problématique similaire avec les utilisateurs qu'on importe depuis une autre base de données, Mandragore. On importe des utilisateurs depuis des bases de données qu'on voudra a terme supprimer.

On a créé une classe MandragoreProfile dans laquelle on stocke le mot de passe Mandragore d'un utilisateur:
https://github.com/erudit/eruditorg/blob/86856741fc92c3a9839a37ad8ec62083069c0cf7/erudit/erudit/models/core.py#L114

Ensuite, on crée un one-to-one de User à MandragoreProfile
https://github.com/erudit/eruditorg/blob/86856741fc92c3a9839a37ad8ec62083069c0cf7/erudit/erudit/models/core.py#L117

Je ferais la même chose ici. Je créerais un objet, IndividualSubscriptionProfile, qui contient le mot de passe de la base de données AbonnementsIndividuels et je ferais le lien one-to-one avec User. Comme ça, si on a un utilisateur qui est à la fois dans Mandragore et Abonnements, on pourrait plus facilement gérer ce cas: il aurait deux profils.